contracts/JetStakingV2.sol

```diff
@@ -250,7 +250,7 @@ contract JetStakingV1 is AdminControlled {
         uint256[] memory scheduleTimes,
         uint256[] memory scheduleRewards,
         uint256 tau
-    ) external onlyRole(STREAM_MANAGER_ROLE) {
+    ) external virtual onlyRole(STREAM_MANAGER_ROLE) {
         _validateStreamParameters(
```

Comment:
`safeTransferFrom` in this function should be above all necessary storage upgrade as it transfers user's tokens, to safe guard against re-entrancy

Reply:
anyways as it is a low level call so not very critical